src/control/client/pool.go

```diff
@@ -25,6 +25,7 @@ package client
 
 import (
 	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
```

Comment:
I think this is ok for now but we should look into some of the other uuid packages eventually. I believe there is some concern in the community that the satori uuid package isn't as good about bug fixes as the community fork of it is. I don't know how true that is but since we already use it we can use it for now and look into a replacement later.

Reply:
yes @mjmac mentioned one that supports a different version of UUIDs that he thought might be beneficial for us to move to

Reply:
Yeah I was thinking about using "v6" UUIDs: http://gh.peabody.io/uuidv6/

I don't think they really add much in our case, though. I've used them in the past with DynamoDB as a way to get a unique range key that has time-sortability built in. With regular v4 UUIDs we can still have the time component; they're just not lexicographically sortable.